classes/class-blog-handler-factory.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+namespace EdLTI;
```

Comment:
Name spaces should follow the PSR 4 convention of reflecting the directory

```suggestion
namespace EdLTI\classes;
```